Assets/LeapMotionModules/InteractionEngine/Scripts/InteractionBehaviour.cs

```diff
@@ -283,6 +286,21 @@ public abstract class InteractionBehaviour : MonoBehaviour {
     }
 
     /// <summary>
+    /// Called by InteractionController when the velocity of an object is changed.
+    /// </summary>
+    public virtual void OnVelocityChanged(UnityEngine.Vector3 linearVelocity, UnityEngine.Vector3 angularVelocity)
```

Comment:
Unfortunately I am not accumulating velocities, I have to clear *everything* and then set them.  There is some unfortunate stuff going on around working with PhysX.